app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -148,16 +148,16 @@ class RecentlyPlayedFragmentTest {
 
   private fun createRecentlyPlayedActivityIntent(profileId: Int): Intent {
     return RecentlyPlayedActivity.createRecentlyPlayedActivityIntent(
-      context,
-      profileId
+      context = context,
+      internalProfileId = profileId
     )
   }
 
   @Test
   fun testRecentlyPlayedTestActivity_clickOnToolbarNavigationButton_closeActivity() {
     activityTestRule.launchActivity(
       createRecentlyPlayedActivityIntent(
-        internalProfileId
+        profileId = internalProfileId
```

Comment:
Write it as `internalProfileId = internalProfileId` everywhere.

Reply:
Done!